packages/jobs/lib/runner/fleet.ts

```diff
@@ -1,8 +1,23 @@
-import type { NodeProvider } from '@nangohq/fleet';
 import { Fleet } from '@nangohq/fleet';
 import { envs } from '../env.js';
 import { localNodeProvider } from './local.js';
+import { renderNodeProvider } from './render.js';
 
-const nodeProvider: NodeProvider | undefined = envs.RUNNER_TYPE === 'LOCAL' ? localNodeProvider : undefined;
+const defaultNodeConfig = {
+    image: 'nangohq/nango-runner',
+    cpuMilli: 500,
+    memoryMb: 512,
+    storageMb: 20000
+};
 
-export const runnersFleet = new Fleet({ fleetId: 'nango_runners', nodeProvider });
+const fleetId = 'nango_runners';
+export const runnersFleet = (() => {
+    switch (envs.RUNNER_TYPE) {
+        case 'LOCAL':
+            return new Fleet({ fleetId, nodeSetup: { nodeProvider: localNodeProvider, defaultNodeConfig } });
```

Comment:
Local Runner does not really have default node config, it should be optional no or simply stored in the provider?

Reply:
Good point. There are indeed linked and maybe should be stored in the provider.

Reply:
fixed in 7b86be260